index.d.ts

```diff
@@ -23303,6 +23327,38 @@ declare namespace Octokit {
      */
     title?: string;
   };
+  export type IssuesUpdateParams = {
+    /**
+     * Logins for Users to assign to this issue. Pass one or more user logins to _replace_ the set of assignees on this Issue. Send an empty array (`[]`) to clear all assignees from the Issue. _NOTE: Only users with push access can set assignees for new issues. Assignees are silently dropped otherwise._
+     */
+    assignees?: string[];
+    /**
+     * The contents of the issue.
+     */
+    body?: string;
+
+    issue_number: number;
+    /**
+     * Labels to associate with this issue. Pass one or more Labels to _replace_ the set of Labels on this Issue. Send an empty array (`[]`) to clear all Labels from the Issue. _NOTE: Only users with push access can set labels for issues. Labels are silently dropped otherwise._
+     */
+    labels?: string[];
+    /**
+     * The `number` of the milestone to associate this issue with or `null` to remove current. _NOTE: Only users with push access can set the milestone for issues. The milestone is silently dropped otherwise._
+     */
+    milestone?: number | null;
+
+    owner: string;
+
+    repo: string;
+    /**
+     * State of the issue. Either `open` or `closed`.
+     */
+    state?: "open" | "closed";
+    /**
+     * The title of the issue.
+     */
+    title?: string;
+  };
```

Comment:
Where do all these come from suddenly?

Reply:
This diff is really weird. When I look at this in an editor, there's a new type for `IssuesUpdateParamsDeprecatedAssignee`, which contains all the fields from `IssuesUpdateParams`, but with an `assignee` field with the deprecated jsdoc.

<img width="796" alt="image" src="https://user-images.githubusercontent.com/2539016/66262819-c951b380-e79c-11e9-8e60-9cd811646a35.png">

In this diff though it only shows `assignee` and `title` on the `IssuesUpdateParamsDeprecatedAssignee` type.

I can look into why this diff doesn't look like it matches the file content.